### DISCUSSION TITLE
Hero 11 Motion Detection

### DISCUSSION THREAD

**Discussion by JohnCash0018** [gopro/labs#99] [Q&A]
Has anyone been able to get the motion detection to work on the Hero 11.  I have installed the firmware update, read the white papers, and tried to make the feature work on QR Control...it does NOT WORK.  Any help would be appreciated.

Thanks!

**Comment by dnewman-gpsw** [comment]
It is works here.  So maybe you are using a corner case camera mode, or just running the wrong QR code.  The default command in this page https://gopro.github.io/labs/control/motion/ works for any video mode I typically use.  
![image](https://user-images.githubusercontent.com/10856084/193321196-f40a6087-e232-4e37-a76d-4cf45c8a66e9.png)

**Reply by JohnCash0018** [reply]
Thank you for the replay!  I finally figured out the issue.  I had the duration set to 3hrs, as soon as I set it to "no duration" it started working perfectly!

**Reply by Bocio** [reply]
@JohnCash0018 where did you have the duration time? I could no t find it on the Motion Detection page/app qrcode
